packages/flutter_tools/templates/plugin/android-java.tmpl/build.gradle.tmpl

```diff
@@ -12,29 +9,36 @@ buildscript {
     }
 }
 
+plugins {
+    id "com.android.library"
+    // Exports "flutter" object.
+    id "dev.flutter.flutter-plugin-gradle-plugin"
+}
+
+group = "{{androidIdentifier}}"
+version = "1.0"
```

Comment:
The `plugins` block can only have a couple of things before it, which includes a buildscript block but does not include these two values.